### PR TITLE
close off brackets correctly  in organising stores page

### PR DESCRIPTION
**Pull request by MaikuB**
Missed that I incorrectly used the square brackets for part of the text

### CODE REVIEW

File: docs/src/guides/organizing-stores.mdx [hunk 1]
```diff
@@ -260,7 +260,7 @@ Example(
 ...
 ```
 
-A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package] of said dependencies can be passed if required.
+A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package) of said dependencies can be passed if required.
```
Comment (pavanpodila) [inline docs/src/guides/organizing-stores.mdx]: Can be changed like so for a simpler language:

```
A side effect of this is that writing widget tests become easier, as the dependencies are well known. We could then pass mocks (e.g. using the [mockito][2] package) of said dependencies, if required.
```

Comment (MaikuB) [reply docs/src/guides/organizing-stores.mdx]: Thanks. Reworded it a bit now

**Comment by MaikuB** [comment]
@pavanpodila one thing I've noticed is that the references don't seem to be appearing on the bottom anymore. I'm not sure what's changed there so may need your help with that

File: docs/src/guides/organizing-stores.mdx [hunk 2]
```diff
@@ -260,7 +260,7 @@ Example(
 ...
 ```
 
-A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package] of said dependencies can be passed if required.
+A side effect of this is that writing widget tests become easier as well. Since the widget's dependencies have been made explicit, we could pass mocks (e.g. using the [mockito][2] package) of said dependencies if required.
```
Comment (pavanpodila) [inline docs/src/guides/organizing-stores.mdx]: a comma is needed after "said dependencies"

**Comment by pavanpodila** [comment]
> @pavanpodila one thing I've noticed is that the references don't seem to be appearing on the bottom anymore. I'm not sure what's changed there so may need your help with that

I see..will take a look after we merge

File: docs/src/guides/organizing-stores.mdx (hunk 2 above)
Comment (MaikuB) [reply docs/src/guides/organizing-stores.mdx]: It's not said with a pause to warrant a comma. A similar usage would be, say, "you can call me if needed"

Comment (pavanpodila) [reply docs/src/guides/organizing-stores.mdx]: got it

**Pull request merged**